imagetool/src/main/resources/docker-files/Update_Image.mustache

```diff
@@ -103,13 +30,12 @@ USER {{userid}}
         RUN DOMAIN_PARENT=$(dirname {{{domain_home}}}) \
         && mkdir -p $DOMAIN_PARENT \
         && chown {{userid}}:{{groupid}} $DOMAIN_PARENT \
-        && chmod g+w $DOMAIN_PARENT \
-        && mkdir -p {{{wdt_model_home}}} \
-        && chmod g+w {{{wdt_model_home}}}
+        && mkdir -p {{{wdt_model_home}}}
```

Comment:
We have two mkdir -p {{{wdt_model_home}}}  before, but what if there is none?   Will the COPY automatically create the structures?

Reply:
it looks to me like there is still a `mkdir -p l{{{wdt_model_home}}}` line on line 33.

Reply:
mkdir -p {{{wdt_model_home}}}
This mkdir is on line 33.  I did not actually modify this line.  I deleted the two chmod's because of the new chmod that you added after the COPY's.